### PR TITLE
Add end-points to fetch one plan or its prices.

### PR DESCRIPTION
**Pull request by eprudnikov**
This PR adds two end-points as a refinement of `/products/{productAnchor}/plans`:
- `/products/{productAnchor}/plans/{planAnchor}`. The response model is exactly the same but represents only one plan.
- `/products/{productAnchor}/plans/{planAnchor}/prices`. The product and add-on prices.

Additionally, each of the end-points now has two common optional parameters:
- currency
- (billing) interval

### CODE REVIEW

File: hub/hub-api.yaml [hunk 1]
```diff
@@ -2973,8 +3082,15 @@ components:
         intervalCount:
           type: integer
           format: int64
-        plan:
-          $ref: "#/components/schemas/Plan"
+
+    PlanPrice:
```
Comment (eprudnikov) [inline hub/hub-api.yaml:3086]: I decided to extract it as a separate model to allow including or extending the price model without its plan.

File: hub/hub-api.yaml [hunk 2]
```diff
@@ -610,6 +616,100 @@ paths:
               schema:
                 $ref: "#/components/schemas/ResponseError"
 
+  /products/{productAnchor}/plans/{planAnchor}:
```
Comment (eprudnikov) [inline hub/hub-api.yaml:619]: @aehrenthal , please note that it implies referring to the plan by its anchor, not id. It helps to fetch the data even if you don't know the id. Shamil wrote me that you will have access to it.

**Review by aehrenthal — APPROVED** [review]

**Comment by eprudnikov** [comment]
@olyavertwist , please post any feedback you have. I'll do the necessary correction (if any) as a separate request.

File: hub/hub-api.yaml [hunk 3]
```diff
@@ -610,6 +616,100 @@ paths:
...
+    get:
+      summary: Get subscription plans
```
Comment (olyavertwist) [inline hub/hub-api.yaml:621]: summary: Retrieve plan details

File: hub/hub-api.yaml [hunk 4]
```diff
@@ -610,6 +616,100 @@ paths:
...
+      description: Use this endpoint to see the plans available for a certain product. You'll also see the plan that is active now.
```
Comment (olyavertwist) [inline hub/hub-api.yaml:622]: description: Use this endpoint to see specific plan details. Provide a `productAnchor` and `planAnchor` in the path.

File: hub/hub-api.yaml [hunk 5]
```diff
@@ -610,6 +616,100 @@ paths:
...
+        default:
+          description: Bad request, security violation, or internal server error.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ResponseError"
+
+  /products/{productAnchor}/plans/{planAnchor}/prices:
+    get:
+      summary: Get subscription plans
```
Comment (olyavertwist) [inline hub/hub-api.yaml:667]: summary: List prices

File: hub/hub-api.yaml [hunk 6]
```diff
@@ -610,6 +616,100 @@ paths:
...
+      description: Use this endpoint to see the plans available for a certain product. You'll also see the plan that is active now.
```
Comment (olyavertwist) [inline hub/hub-api.yaml:668]: description: Use this endpoint to list prices for a certain product and plan. Provide a `productAnchor` and `planAnchor` in the path.

File: hub/hub-api.yaml [hunk 7]
```diff
@@ -610,6 +616,100 @@ paths:
...
+            $ref: '#/components/schemas/PriceInterval'
+        - name: currency
+          in: query
+          description: The currency of the prices.
+          required: false
+          schema:
+            $ref: '#/components/schemas/Currency'
+      responses:
+        200:
+          description: The list of prices associated with the plan. It includes prices for the product and the add-ons.
```
Comment (olyavertwist) [inline hub/hub-api.yaml:699]: description: Returns the list of prices associated with the plan, including prices for the product and its add-ons.

Comment (eprudnikov) [reply hub/hub-api.yaml:699]: Thanks, Olly, for noticing. It missed those copy-paste artifacts. I applied the fixes in [the following PR](https://github.com/inperium-corp/inperium-openapi/pull/324).

**Pull request merged**